vireoSDK.props

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <PropertyGroup>
+        <vireo_InstallDir>$(MSBuildThisFileDirectory)..\nipkgcore\</vireo_InstallDir>
```

Comment:
This looks sketchy to me. We are making a nuget package for Vireo. It should be generic and just include the vireo.exe as content files. This is assuming it is being used in the build system and assuming some filesystem layout.